### ISSUE TITLE
"What's This?" Implementation

### ISSUE THREAD

**Issue by ariel** [basilleaf/luti#21]
Ability for a user to share their current screen in both hyperlink location on the grid and "screenshot" of what they're looking at. This is a stretch goal. Assigned to Matt.

**Comment by mattb** [comment]
Depends on #22

**Comment by ariel** [comment]
Merging in the "More Info" requirement that was at #21: Ability for a user to click on a microbe video to get more info about its scale, what type of microbe, what environment is was sampled from, etc.
